zstd_compression_format.md

```diff
@@ -97,6 +97,42 @@ to decode all concatenated frames in their sequential order,
 delivering the final decompressed result as if it was a single content.
 
 
+Skippable Frames
```

Comment:
I moved "skippable frames" from the middle where they did not fit. For me they can be at the beginning because the section is short and easy to read and it's related to "Frame Concatenation". I can move them to the end if you wish.